### PR TITLE
Clean up errors flagged by IDE and remove final/target

### PR DESCRIPTION
**Pull request by iambic69**
Paul Balfour set me this guide recently.  I enjoyed working through it.  Everything worked fine from the command line, but I noticed some issues when I viewed the source in my IDE.

### CODE REVIEW

**Comment by iambic69** [comment]
Hi @enozcan.  I saw you removed final/target via another PR.  Are you okay with the other changes I put forward in this one?

**Comment by enozcan** [comment]
Hi @iambic69 

Thanks for noticing about these issues.
Instead of changing the `package` statements in java files, I've changed only the package name and the inconsistency is now removed with #28 .

Also, can you inform me about the point of `Qualifier` annotation for HazelcastInstance. Is that a must when using Autowired fields?

**Comment by iambic69** [comment]
Hi @enozcan

Without it, IntelliJ IDEA gives an error "Could not autowire. There is more than one bean of 'HazelcastInstance' type."  I did a little digging, and it looks like it's a long-standing limitation of IntelliJ's support for Spring auto-configuration.  See:

- [IntelliJ incorrectly showing error: "Could not autowire. There is more than one bean of 'OAuth2ClientContext' type.](https://youtrack.jetbrains.com/issue/IDEA-176467)
- [Spring Boot: Support @EnableAutoConfiguration](https://youtrack.jetbrains.com/issue/IDEA-139669)

Basically, IntelliJ isn't able to determine which of the two beans provided in `org.springframework.boot.autoconfigure.hazelcast.HazelcastServerConfiguration` should be used.  Since both beans have the same name, I can't see why adding `Qualifier` would make the problem go away, but it does.

Your call whether you want to add it to the guide as a workaround for this issue.  It didn't seem to have any other side-effects.

**Comment by enozcan** [comment]
Hi @iambic69 

Thanks for the answer. I will check the `Qualifier` issue again.
And since the problems you mentioned in this PR is resolved in #26 and #28 I'm closing the PR.
